pkg/render/logstorage.go

```diff
@@ -1136,6 +1136,10 @@ func (es elasticsearchComponent) kibanaCR() *kbv1.Kibana {
 			"rewriteBasePath": true,
 		},
 		"elasticsearch.ssl.certificateAuthorities": []string{"/usr/share/kibana/config/elasticsearch-certs/tls.crt"},
+		"tigera": map[string]interface{}{
+			"enabled":        true,
+			"licenseEdition": "enterpriseEdition",
```

Comment:
What is this licenseEdition? Should this be based on something read from the LicenseKey?

Reply:
This is for updating the branding/logos in Kibana. If this is something that can be inferred from LicenseKeys then that would be better here.

Reply:
Note that this is something that is in the management cluster and I am not fully aware of how we do licensing in Calico Cloud.